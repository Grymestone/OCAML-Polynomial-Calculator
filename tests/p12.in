(2a + 3a^6 + 1a^2)^4 - (4a + 3a^2)
