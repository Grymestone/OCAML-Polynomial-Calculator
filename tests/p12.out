(((((2*a)+(3*(a)^6))+(1*(a)^2)))^4-((4*a)+(3*(a)^2)))
108x^20 + 81x^24 + 216x^19 + 54x^16 + 216x^15 + 216x^14 + 12x^12 + 72x^11 + 144x^10 + 96x^9 + 1x^8 + 8x^7 + 24x^6 + 32x^5 + 16x^4 + -3x^2 + -4x^1